src/App.tsx

```diff
@@ -32,8 +32,10 @@ const flags = {
     argonaut: isHostedApiEnabled('argonaut', true),
     benefits: isHostedApiEnabled('benefits', true),
     claims: isHostedApiEnabled('claims', true),
+    disability_rating: isHostedApiEnabled('disability_rating', true),
     facilities: isHostedApiEnabled('facilities', true),
-    loan_guarantees: isHostedApiEnabled('loan_guarantees', false)
+    loan_guarantees: isHostedApiEnabled('loan_guarantees', false),
+    service_history: isHostedApiEnabled('service_history', true)
```

Comment:
```suggestion
    service_history: isHostedApiEnabled('service_history', true),
```

I think we might want to add to our linter, but its easier for having to redo alpha sorts latter if we leave trailing commas.